mirrord/cli/src/config.rs

```diff
@@ -156,6 +156,9 @@ pub(super) enum Commands {
     #[command(name = "port-forward")]
     PortForward(Box<PortForwardArgs>),
 
+    /// Manage database branching.
+    #[command(name = "db-branches")]
+    DbBranches(Box<DbBranchesArgs>),
```

Comment:
super nit: add a newline after this to make it consistent with other fields of the struct.